packages/admin/src/common/toolbar/Toolbar.tsx

```diff
@@ -0,0 +1,27 @@
+import { Paper, Toolbar as MUIToolbar } from "@material-ui/core";
```

Comment:
In my opinion, all Material-UI renames should be imported with lowercase characters, like `MuiToolbar`

Reply:
ok i don't really care about that, but i can rename it for you ;-)